### PR TITLE
NOISSUE - Add default subscription nodeID and Interval ENVAR

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: opcua/gopcua/subscribe.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (c client) Subscribe(cfg opcua.Config) error {
 	defer oc.Close()
 
 	sub, err := oc.Subscribe(&opcuaGopcua.SubscriptionParameters{
-		Interval: 2000 * time.Millisecond,
+		Interval: 1000 * time.Millisecond,
```
Comment (bbokun) [inline opcua/gopcua/subscribe.go]: Does it make sense to use an env variable from outside of the image?

File: opcua/api/endpoint.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ func browseEndpoint(svc opcua.Service) endpoint.Endpoint {
 			return nil, err
 		}
 
-		nodeID := fmt.Sprintf("ns=%s;%s", req.Namespace, req.Identifier)
+		nodeID := fmt.Sprintf("%s;%s", req.Namespace, req.Identifier)
```
Comment (bbokun) [inline opcua/api/endpoint.go]: Didn't we say that when the req.Namespace and req.Identifier are empty, the defaults of ns=1, i=84 are automatically used, as the user definitely can not know this information.

**Review by bbokun — CHANGES_REQUESTED** [review]

File: opcua/gopcua/subscribe.go (hunk 1 above)
Comment (drasko) [reply opcua/gopcua/subscribe.go]: In any case, move constants to the top. But yes, this should probably be configurable via ENV var.

**Review by drasko — CHANGES_REQUESTED** [review]

File: cmd/opcua/main.go [hunk 3]
```diff
@@ -46,6 +47,7 @@ const (
 
 	envHTTPPort       = "MF_OPCUA_ADAPTER_HTTP_PORT"
 	envLogLevel       = "MF_OPCUA_ADAPTER_LOG_LEVEL"
+	envOPCIntervalMs  = "MF_OPCUA_ADAPTER_INTERVAL_MS"
```
Comment (drasko) [inline cmd/opcua/main.go:50]: Should this be added to README?

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by darkodraskovic — DISMISSED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**